docs/running.rst

```diff
@@ -47,8 +47,15 @@ Options:
   is ignored for any other target.
 
 -o, \-\-output *directory*
-  This option takes one argument, which is the directory where output should
-  be saved. The default is the current directory.
+  Sets the directory where the output should be saved. This defaults to the current directory if not specifed.
```

Comment:
What is the current directory? That of the Solidity source file or that of the Solang binary?

Reply:
Neither, a file does not have current directory, only a process. How about calling it the current working directory?

Reply:
That sounds good.